recipes/qcatch/meta.yaml

```diff
@@ -0,0 +1,48 @@
+{% set name = "qcatch" %}
+{% set version = "0.1.2" %}
+
+package:
+  name: "{{ name }}"
+  version: "{{ version }}"
+
+source:
+  url: https://files.pythonhosted.org/packages/source/q/qcatch/qcatch-{{ version }}.tar.gz
+  sha256: 1724f82b9c22b3b7c9986a7c519fef7273dfe0451bb49cf8621032319ac2667e
+
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed -vv"
+  run_exports:
+    - {{ pin_subpackage('qcatch', max_pin='x.x') }}
+
+requirements:
+  host:
+    - python =3.12.9
```

Comment:
Any particular reason why this very specific version is needed?  Consider updating/testing `qcatch` with different versions